aten/src/ATen/native/mps/operations/Pad.mm

```diff
@@ -15,27 +15,28 @@
   const int padding_size = (int) padding.size();
   const int padding_dim = padding_size / 2; // either 1D, 2D, or 3D
 
+  TORCH_CHECK(input_.numel() != 0, "MPS currently doesn't support zero-element input.")
```

Comment:
Added a check to prevent users from getting an uninitialized empty output when the padding mode is constant and the number of input elements is 0.